assets/js/formbuilder.js

```
@@ -839,6 +941,21 @@
                     if (hasDef) line += ', ' + phpStr(item.defaultValue);
                     break;
                 }
+                case 'togglecheckbox':
+                case 'colorswatch': {
+                    line += ', ' + colorSwatchArray(parseColorSwatches(item.options));
+                    var hasDefaultColorValue = !!item.defaultValue;
+                    if (attrPhp || hasDefaultColorValue) line += ', ' + (attrPhp || 'null');
+                    if (hasDefaultColorValue) line += ', ' + phpStr(item.defaultValue);
+                    break;
+                }
+                case 'togglecheckbox': {
```

Biome (2.4.15)

[error] Duplicate case label.

(lint/suspicious/noDuplicateCase)